src/Promitor.Tests.Unit/Validation/Scraper/Metrics/Sinks/StatsDMetricSinkValidationStepTests.cs

```diff
@@ -87,6 +88,38 @@ public void Validate_StatsDWithoutHost_Fails(string host)
             PromitorAssert.ValidationFailed(validationResult);
         }
 
+        [Theory]        
+        [InlineData(null)]
+        public void Validate_StatsDWithGenevaFormatWithoutGenevaConfiguration_Fails(GenevaConfiguration genevaConfiguration)
```

Comment:
Let's also add a case where account is specified but namespace is not, and vice versa